jekyll.yml

```diff
@@ -65,7 +65,7 @@ ee_product_name: Kong Gateway Enterprise
 konnect_short_name: Konnect
 mesh_product_name: Kong Mesh
 kic_product_name: Kong Ingress Controller
-konnect_saas: Kong Konnect
+konnect_product_name: Kong Konnect
 
```

Comment:
It might be helpful to add "Konnect Reference Platform" as a variable as well. `ref_plat` if you want it short.